scripts/restore

```diff
@@ -55,14 +49,16 @@ sudo chown -R root:root $final_path
 sudo chown -R www-data:root $final_path/{data,tmp}
 sudo chmod -R 700 $final_path/{data,tmp}
 
-# Restore conf files
-sudo cp -a "./conf/nginx.conf" $conf
-
-# Reload Nginx
-sudo service nginx reload
+# Restore configuration files
+sudo cp -a ./conf/nginx.conf "${nginx_conf}"
 
 # Set ssowat config
+ynh_app_setting_set "$app" is_public "$is_public"
 if [ "$is_public" = "Yes" ];
 then
-  sudo yunohost app setting $app unprotected_uris -v "/"
+  ynh_app_setting_set "$app" unprotected_uris "/"
 fi
+
+# Reload service
+sudo systemctl reload nginx
+sudo yunohost app ssowatconf
```

Comment:
not needed.
Must be already done by YunoHost after this the execution of this script.